data/formats-data.js

```diff
@@ -6431,13 +6458,14 @@ exports.BattleFormatsData = {
 		tier: "RU",
 	},
 	jangmoo: {
+		randomBattleMoves: ["dragondance", "outrage", "dragonclaw", "earthquake", "substitute"],
```

Comment:
+ brick break

Reply:
why?

Reply:
oh right, this one isnt dragon type
nvm then lmao